addon/models/user.js

```diff
@@ -3,6 +3,7 @@ import DS from 'ember-data';
 import OsfModel from 'ember-osf/models/base';
 
 export default OsfModel.extend({
+    full_name: DS.attr('string'),
```

Comment:
These fields should really be camelCased (my fault)-- can you please change them?